code/modules/projectiles/guns/projectile/sniper.dm

```diff
@@ -16,23 +15,17 @@
 	zoom_amt = 7 //Long range, enough to see in front of you, but no tiles behind you.
 	slot_flags = SLOT_BACK
 
-
-/obj/item/weapon/gun/projectile/sniper_rifle/update_icon()
-	if(magazine)
-		icon_state = "sniper-mag"
-	else
-		icon_state = "sniper"
-
-
 /obj/item/weapon/gun/projectile/sniper_rifle/syndicate
 	name = "syndicate sniper rifle"
 	desc = "Syndicate flavoured sniper rifle, it packs quite a punch, a punch to your face"
 	pin = /obj/item/device/firing_pin/implant/pindicate
 	origin_tech = "combat=8;syndicate=4"
 
-
-
-
+/obj/item/weapon/gun/projectile/sniper_rifle/syndicate/update_icon()
```

Comment:
uh? you're making the sniper only update it's icon if it's syndicate? that's stupid stop.